Tests/PostgresNIOTests/New/PSQLRowStreamTests.swift

```diff
@@ -19,20 +19,29 @@ final class PSQLRowStreamTests: XCTestCase {
         
         XCTAssertEqual(try stream.all().wait(), [])
         XCTAssertEqual(stream.commandTag, "INSERT 0 1")
+
+        XCTAssertNoThrow(try stream.drain().wait())
```

Comment:
all these tests are covering real-world use-cases.

we hand the postgres-sequence over to users. If they have fully consumed the rows, a `drain()` call should simply just do nothing. Not crash or ....